File: calculadora.py
def calcular_roi(ganho, investimento):
    return (ganho - investimento) / investimento

investimento = float(input("Digite o valor do investimento: "))

if investimento == 0:
    print("Erro: O investimento não pode ser 0.")
else:
    ganho = float(input("Digite o valor do ganho: "))
    
    roi = calcular_roi(ganho, investimento)
    print(f"ROI: {roi:.2f} ({roi * 100:.2f}%)")

File: calculadora.cpp
#include <iostream>
using namespace std;

double calcularROI(double ganho, double investimento) {
    return (ganho - investimento) / investimento;
}

int main() {
    double investimento, ganho;

    cout << "Digite o valor do investimento: ";
    cin >> investimento;

    if (investimento == 0) {
        cout << "Erro: o investimento nao pode ser 0." << endl;
    } else {
        cout << "Digite o valor do ganho: ";
        cin >> ganho;

        double roi = calcularROI(ganho, investimento);
        cout << "ROI: " << roi << " (" << roi * 100 << "%)" << endl;
    }

    return 0;
}